importer/cusource/build_cu.go

```diff
@@ -0,0 +1,99 @@
+package cusource
+
+import (
+	"database/sql"
+	"encoding/json"
+	"github.com/Bnei-Baruch/mdb/api"
+	"github.com/Bnei-Baruch/mdb/common"
+	"github.com/Bnei-Baruch/mdb/models"
+	"github.com/Bnei-Baruch/mdb/utils"
+	log "github.com/Sirupsen/logrus"
+	"github.com/volatiletech/sqlboiler/boil"
+	"github.com/volatiletech/sqlboiler/queries"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+	"gopkg.in/volatiletech/null.v6"
+	"time"
+)
+
+func BuildCUSources(mdb *sql.DB) ([]*models.Source, []*models.ContentUnit) {
+
+	rows, err := queries.Raw(mdb,
+		`SELECT cus.source_id FROM content_units cu
+			INNER JOIN content_units_sources cus ON cus.content_unit_id = cu.id
```

Comment:
content_units_sources is wrong here. See comment above about the link of source and its CU (property)